<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestPassing</class>
 <widget class="QWidget" name="TestPassing">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>621</width>
     <height>421</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="Answer4RadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>16</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="Answer1RadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>16</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="Answer2RadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>16</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Answer3Label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>280</y>
      <width>551</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>третий ответ</string>
    </property>
   </widget>
   <widget class="QLabel" name="Answer2Label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>230</y>
      <width>531</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>второй ответ</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Answer3CheckBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>16</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Answer1Label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>180</y>
      <width>541</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>первый ответ</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Answer3RadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>16</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Answer4Label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>330</y>
      <width>541</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>четвертый ответ</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Answer1CheckBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>16</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="Answer2CheckBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>16</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="Answer4CheckBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>16</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="QuestionLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>591</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>вопрос</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="KostilRadioButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>20</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="EndTestButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>30</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Закончить тест</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="DetailedAnswerLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>601</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="CurrentQuestionSpinBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>42</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="TimerLabel">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>60</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
